package.json

```diff
@@ -19,7 +19,8 @@
   "resolutions": {
     "glob": "^9.0.0",
     "optionator": "^0.9.3",
-    "semver": "^7.5.2"
+    "semver": "^7.5.2",
+    "protobufjs": "^7.2.5"
```

Comment:
I had to force upgrade `protobufjs` to pull `7.2.5` or higher version because when I tried the updating `protobufjs` as transitive dependency through the steps outlined in Readme section above, it only updated protobufjs version for 2 of the 3 usage and for some reason `@google-cloud#logging#google-gax` was still pulling `7.2.4`. Hence I force upgraded it.

Commit in which I tried updating it as transitive dependency: https://github.com/DataBiosphere/bard/pull/88/commits/4ea0292c2a0ac6a34d23dbcc51be3fa6dac77bed
Output of `yarn why` for that commit:
```
 % yarn why protobufjs
yarn why v1.22.0
warning ../../../package.json: No license field
[1/4] 🤔  Why do we have the module "protobufjs"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "protobufjs@7.3.2"
info Has been hoisted to "protobufjs"
info Reasons this module exists
   - Hoisted from "@google-cloud#logging#google-gax#proto3-json-serializer#protobufjs"
   - Hoisted from "@google-cloud#logging#google-gax#@grpc#proto-loader#protobufjs"
info Disk size without dependencies: "2.85MB"
info Disk size with unique dependencies: "7.23MB"
info Disk size with transitive dependencies: "7.23MB"
info Number of shared dependencies: 12
=> Found "google-gax#protobufjs@7.2.4" // <==== still pulling vulnerable version
info This module exists because "@google-cloud#logging#google-gax" depends on it.
✨  Done in 0.21s.
```